ckanext/stadtzhtheme/descr.yaml

```diff
@@ -31,3 +31,12 @@ gpkg:
 dxf:
     description: Drawing Interchange File Format. Projektion CH1903+ / LV95 (EPSG:2056)
     link: Weitere Informationen zu DXF finden Sie in unserer Rubrik Werkstatt unter [Informationen zu Datenformaten.](https://www.stadt-zuerich.ch/portal/de/index/ogd/werkstatt/dxf.html)
+gdb:
+    description: Geodatabase. Projektion CH1903+ / LV95 (EPSG:2056)
+    link: "Weitere Informationen zu Geodatabase finden Sie auf der Webseite von ESRI: [Was ist eine Geodatabase?](https://desktop.arcgis.com/de/arcmap/10.3/manage-data/geodatabases/what-is-a-geodatabase.htm)."
```

Comment:
Hier haben sich Anführungszeichen eingeschlichen, die ihr in den anderen Einträgen nicht habt. Sind die gewollt, oder können die entfernt werden?

Reply:
Die sind eben nötig, weil wir einen Doppelpunkt im Text haben. Ich lasse das ganze File mal noch kurz durch http://www.yamllint.com/ laufen und aktualisiere alle Einträge.

Reply:
Super, merci für die Erklärung :)